editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.tsx

```diff
@@ -0,0 +1,198 @@
+          variant="scrollable"
+          value={activeTab}
+          onChange={handleChange}
+          aria-label="Vertical tabs example"
+          sx={{ borderRight: 1, borderColor: "divider" }}
+        >
+          {features.map((feature, i) => (
+            <Tab
+              key={`tab-${i}`}
+              value={feature.properties?.label}
+              label={`${schema.type} ${feature.properties?.label}`}
+              {...a11yProps(i)}
+            />
+          ))}
+        </Tabs>
+        {features.map((feature, i) => (
+          <TabPanel
+            key={`tabpanel-${i}`}
+            value={feature.properties?.label}
+            sx={{ width: "100%" }}
+          >
+            <Typography component="h2" variant="h3">
+              {`${schema.type} ${feature.properties?.label}`}
+            </Typography>
+            <Typography variant="body2" gutterBottom>
+              {`${feature.geometry.type}`}
+              {feature.geometry.type === "Point"
+                ? ` (${feature.geometry.coordinates.map((coord) =>
+                    coord.toFixed(5),
+                  )})`
+                : ` (area ${
+                    feature.properties?.["area.squareMetres"] || 0
+                  } m²)`}
+            </Typography>
+            <SchemaFields
+              schema={schema}
+              activeIndex={activeIndex}
+              formik={formik}
+            />
+          </TabPanel>
+        ))}
+      </TabContext>
+    </Box>
+  );
+};
+
+const Root = () => {
+  const { validateAndSubmitForm, mapAndLabelProps } = useMapAndLabelContext();
+  const {
+    title,
+    description,
+    info,
+    policyRef,
+    howMeasured,
+    drawColor,
+    drawType,
+    schemaName,
+  } = mapAndLabelProps;
+
+  const teamSettings = useStore.getState().teamSettings;
+  const passport = useStore((state) => state.computePassport());
+
+  const [features, setFeatures] = useState<Feature[] | undefined>(undefined);
+
+  useEffect(() => {
+    const geojsonChangeHandler = ({ detail: geojson }: any) => {
+      if (geojson["EPSG:3857"]?.features) {
+        setFeatures(geojson["EPSG:3857"].features);
+      } else {
+        // if the user clicks 'reset' on the map, geojson will be empty object, so set features to undefined
+        setFeatures(undefined);
+      }
+    };
+
+    const map: any = document.getElementById("map-and-label-map");
+
+    map?.addEventListener("geojsonChange", geojsonChangeHandler);
+
+    return function cleanup() {
+      map?.removeEventListener("geojsonChange", geojsonChangeHandler);
+    };
+  }, [setFeatures]);
+
+  return (
+    <Card handleSubmit={validateAndSubmitForm} isValid>
+      <CardHeader
+        title={title}
+        description={description}
+        info={info}
+        policyRef={policyRef}
+        howMeasured={howMeasured}
+      />
+      <FullWidthWrapper>
+        <MapContainer environment="standalone">
+          {/* @ts-ignore */}
+          <my-map
```

Comment:
nit/follow-up: We need FindProperty again to display this in-Editor, we should have a GraphError component here 👍

Reply:
Yep now implemented !